<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAluno</class>
 <widget class="QDialog" name="DialogAluno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNome">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Curso:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCurso">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sistemas de Informação</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Licenciatura em Computação</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Engenharia Ambiental</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Agrimensura</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Agronomia</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Engenharia de Pesca</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Engenharia Florestal</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matricula:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTelefone">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMatricula">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmail">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineNome</tabstop>
  <tabstop>comboBoxCurso</tabstop>
  <tabstop>lineMatricula</tabstop>
  <tabstop>lineTelefone</tabstop>
  <tabstop>lineEmail</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
